examples/tutorial/apps/taxi_player/main.py

```diff
@@ -74,14 +74,14 @@ def animate_update():
     slider.value = year
 
 def animate():
-    if button.label == '► Play':
-        button.label = '❚❚ Pause'
+    if button.label == '\u25B6 Play':
```

Comment:
This doesn't seem like a fix, but a workaround for something broken elsewhere. Can you say what was broken? In #109 I found the app doesn't work on windows, and suspected/guessed the problem probably came from bokeh.

Reply:
See https://github.com/ioam/jupytercon2017-holoviews-tutorial/issues/5

Reply:
Ok, seems like the same problem. I think this change should only be considered a temporary fix. While it doesn't matter much just for play/pause buttons in this app, people are going to struggle to read and write words using hex codes...